v63002/u_inref/gtm8711.csh

```diff
@@ -0,0 +1,21 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests GDE appropriately maintains the return status when invoked from the shell
```

Comment:
Want to see the below release note also tested.

Also, the GT.M help facility handles an EOF on $PRINCIPAL; previously an EOF produced an internal error and generated an associated context dump file. (GTM-8711)

Reply:
This comment is about the following release note.

*Also, the GT.M help facility handles an EOF on $PRINCIPAL; previously an EOF produced an internal error and generated an associated context dump file. (GTM-8711)*

Looks like this part of the change does not work as documented. At least when the terminal is the principal device, killing the terminal does produce a gtmhelpdmp.ZSHOW_DMP_* file in V63001A as well as V63002 so not sure what this release note means. So ignore my comment asking to test this change.